packages/documentation/src/utils/component-properties.ts

```diff
@@ -0,0 +1,5 @@
+export function definedProperties(properties: Record<string, unknown>) {
+  return Object.fromEntries(
+    Object.entries(properties).filter(([_, value]) => typeof value !== 'undefined')
+  );
```

Comment:
```suggestion
  return Object.values(properties).filter(value => typeof value !== 'undefined')
  );
```

I'm not 100% sure if it's doing the same thing as your logic, but it seems so to me.
Please check and ignore if this is not the case.

Reply:
I need `definedProperties` to return an object with only properties that have a defined value.
The solution you proposed returns an array of set values ​​but the keys are lost.